lib/generators/boring/whenever/install/install_generator.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Boring
+  module Whenever
+    class InstallGenerator < Rails::Generators::Base
+      desc "Adds whenever gem to the application"
```

Comment:
Let's update the description to "Adds whenever gem to the application for managing cron jobs"